mysql/my.cnf

```diff
@@ -1,5 +1,8 @@
 [mysqld]
+user=mysql
+explicit_defaults_for_timestamp
```

Comment:
I wasn't aware of this option until just now, but it looks like it disables some deprecated but currently default behavior. Do we know that Phabricator doesn't rely on the default behavior?